src/assets/toolkit/styles/templates/_print.scss

```diff
@@ -74,11 +74,22 @@
 .hide-for-print {
   @media print {
     display: none;
-  }  
+  }
 }
 
 .show-for-print {
   @media print {
     display: block;
-  }  
+  }
+}
+
+@media print {
+  [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak, .ng-hide:not(.ng-hide-animate) {
+   display: block !important;
```

Comment:
@kaptron I see. We can do that for all the stuff that uses `ng-text-truncate`, but some doesn't use `ng-text-truncate` and uses the `ng-show` and manually sets up the toggle E.g. see `app/assets/javascripts/listings/templates/listing/_what_to_expect.html.slim`

are you saying we convert the manually truncated stuff to all use ng-text-truncate, and update the `ng-text-truncate` css?